### DISCUSSION TITLE
Folders to organize Supporter Tiers

### DISCUSSION THREAD

**Discussion by ssiwm** [gameserverapp/Platform#507] [Suggestions]
I have a request to add folders to organize Supporter Tiers similarly to Shop Packs

**Comment by zerafox** [comment]
Not sure what this folder bit is talking about, only thing I can correlate that with is the Labels which can be used to sort and filter in-game and on-site.

This same function would be wonderful for Supporter Tiers too!

**Comment by DieselEffect** [comment]
This would be a useful feature as we could separate ARK: Survival Evolved items from ARK: Survival Ascended items. This would be strictly organizational from a server manager's perspective.

**Comment by mvaessen** [comment]
Soon (tm)!

**Comment by mvaessen** [comment]
Thanks for your suggestion! The folders are not the same as for Shop packs, at least not what the initial post is referring too.
This is now available on the dashboard
